lib/Runtime/Library/JavascriptMap.h

```diff
@@ -12,15 +12,49 @@ namespace Js
         typedef JsUtil::KeyValuePair<Field(Var), Field(Var)> MapDataKeyValuePair;
         typedef MapOrSetDataNode<MapDataKeyValuePair> MapDataNode;
         typedef MapOrSetDataList<MapDataKeyValuePair> MapDataList;
-        typedef JsUtil::BaseDictionary<Var, MapDataNode*, Recycler, PowerOf2SizePolicy, SameValueZeroComparer> MapDataMap;
+        typedef JsUtil::BaseDictionary<Var, MapDataNode*, Recycler> SimpleVarDataMap;
```

Comment:
Why not use RecyclerPointerComparer here? (Same question for Set)

Reply:
Never mind, I see that simple vars also include tagged ints.

---
In reply to: [174860448](https://github.com/Microsoft/ChakraCore/pull/4816#discussion_r174860448) [](ancestors = 174860448)